test/integration/targets/junos_banner/tasks/netconf.yaml

```diff
@@ -2,6 +2,8 @@
   find:
     paths: "{{ role_path }}/tests/netconf"
     patterns: "{{ testcase }}.yaml"
+  connection: local
+  delegate_to: localhost
```

Comment:
`delegate_to: localhost` is redundant with `connection: local` set here and below.

Reply:
fixed